Data/SeedData.cs

```diff
@@ -11,20 +11,63 @@ public static void Initialize(IServiceProvider serviceProvider)
             if (context == null || context.Season == null) throw new ArgumentNullException("No season in context");
 
             if (context.Season.Any()) return; // already seeded
-            
+
+            var season1 = new Season
+            {
+                Title = "2021/2022",
+                Description = "Ročník 2021/2022",
+                ID = 0
```

Comment:
Season 'ID' by mohlo být 'Id' jako v ostatních modelech.

S tou explicitní definicí ID = 0 a ID = 1 to nejsem schopný rozjet:
`Unhandled exception. System.InvalidOperationException: The instance of entity type 'Season' cannot be tracked because another instance with the key value '{ID: 1}' is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached.`